src/api/locales/queries/getLocales/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import Locale from '@models/Locale'
+import { AUTHENTICATION_ERROR_NO_USER } from '@constants/errors'
+
+type Context = {
+  request: express$Request
+}
+
+const resolver = async (_: any, args: Array<any>, { request }: Context) => {
```

Comment:
Args aren’t an array. `args: any` works.
I think if you don’t use them you could also do `_: any` like for the first argument.